test/test_sparse.py

```diff
@@ -947,7 +996,7 @@ def test_not_in_place(x):
     def test_add_zeros(self, device, dtype, coalesced):
         def test_shape(sparse_dims, nnz, sizes):
             x, _, _ = self._gen_sparse(sparse_dims, nnz, sizes, dtype, device, coalesced)
-            zeros = torch.zeros(sizes, layout=torch.sparse_coo).to(x.device)
+            zeros = torch.sparse_coo_tensor(sizes).to(x.device)
```

Comment:
nit: could pass device directly to constructor

Reply:
right, fixed.